Makefile

```diff
@@ -62,15 +62,15 @@ vendor:
 	go mod vendor
 
 .PHONY: build
-build: vendor fmt vet ## Build manager binary.
+build: vendor fmt vet build-hypershift ## Build manager binary.
```

Comment:
Please make sure the DownStream build can support this.

Reply:
eh... have you worked with downstream for this kind of issue... I don't know what to do for downstream...